packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -214,6 +214,11 @@ function* doCreateChat(action: ReturnType<typeof createChat>) {
       .map((id) => encodeHashId(id))
       .sort()
       .join(':')
+
+    // Optimistically navigate - if we fail we'll toast
+    yield* put(goToChat({ chatId }))
```

Comment:
Does this fail often on a newly created chat? I know we discussed this but wondering what you observed.